crates/cheatnet/src/runtime_extensions/forge_runtime_extension/mod.rs

```diff
@@ -250,6 +250,17 @@ impl<'a> ExtensionLogic for ForgeExtension<'a> {
                     .stop_spoof(target);
                 Ok(CheatcodeHandlingResult::Handled(vec![]))
             }
+            "replace_bytecode" => {
+                let contract = input_reader.read_felt().into_();
+                let class = input_reader.read_felt().into_();
+
+                extended_runtime
+                    .extended_runtime
+                    .extension
+                    .cheatnet_state
+                    .replace_class_for_contract(contract, class);
+                Ok(CheatcodeHandlingResult::Handled(vec![]))
+            }
```

Comment:
I wonder if it wouldn't be bettter to just set the class hash for the address in the state (`extended_runtime .extended_runtime.extended_runtime.hint_handler.state`)

Reply:
It would work only because implementations of `blockifier::state::state_api::State::set_class_hash_at` don't follow interface with return value. it claims that

> Raises an exception if the address is already assigned

but checks only if `contract_address` is zero.